### PR TITLE
[FIX] website_sale_product_minimal_price: combination does not exist message error

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa
TT21362

### CODE REVIEW

File: website_sale_product_minimal_price/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     'name': 'Website Sale Product Minimal Price',
     'summary': 'Display minimal price for products that has variants',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.0.1',
```
Comment (pedrobaeza) [inline website_sale_product_minimal_price/__manifest__.py:6]: Remember that this is done by the bot.

**Review by pedrobaeza — APPROVED** [review]

File: website_sale_product_minimal_price/__manifest__.py (hunk 1 above)
Comment (sergio-teruel) [reply website_sale_product_minimal_price/__manifest__.py:6]: Ok, But should i delete it?

Comment (pedrobaeza) [reply website_sale_product_minimal_price/__manifest__.py:6]: Well, not this time, unless you plan to put other PR over the same module.

Comment (sergio-teruel) [reply website_sale_product_minimal_price/__manifest__.py:6]: ok

**Review by Tardo — APPROVED** [review]
:+1: LGTM

**Pull request merged**